src/webgpu/api/validation/render_pipeline/depth_stencil_state.spec.ts

```diff
@@ -32,6 +32,45 @@ g.test('format')
     t.doCreateRenderPipelineTest(isAsync, !!info.depth || !!info.stencil, descriptor);
   });
 
+g.test('depthCompare')
+  .desc(`The depthCompare in depthStencilState is optional for stencil-only formats.`)
```

Comment:
nit: but required for formats with a depth

Reply:
Done